crates/ink/codegen/src/generator/trait_def/call_builder.rs

```diff
@@ -219,6 +233,16 @@ impl CallBuilder<'_> {
                 }
             }
 
+            impl<E, AccountId> ::core::convert::From<AccountId> for #call_builder_ident<E>
```

Comment:
:heart:

Reply:
Later we could add the same for `ContractRef` generated for the contract itself, but it requires adding a generic to add `AccountIdGuard` bound=)